tests/functional/spec/answer_action_redirect_to_list_list_add_question.js

```diff
@@ -0,0 +1,120 @@
+const helpers = require('../helpers');
+const ListCollectorPage = require('../generated_pages/answer_action_redirect_to_list_add_question/list-collector.page');
+const ListCollectorAddPage = require('../generated_pages/answer_action_redirect_to_list_add_question/list-collector-add.page');
+const ListCollectorRemovePage = require('../generated_pages/answer_action_redirect_to_list_add_question/list-collector-remove.page');
+const AnyoneElseLiveHereBlock = require('../generated_pages/answer_action_redirect_to_list_add_question/anyone-else-live-here-block.page');
+
+function checkPeopleInList(peopleExpected) {
+  let chain = browser.waitForVisible(ListCollectorPage.listLabel(1)).should.eventually.be.true;
+
+  for (let i = 1; i <= peopleExpected.length; i++) {
+    chain = chain.then(() => {
+      return browser.getText(ListCollectorPage.listLabel(i)).should.eventually.equal(peopleExpected[i - 1]);
+    });
+  }
+
+  return chain;
+}
+
+describe('Answer Action: Redirect To List Add Question', function () {
+
+  describe('Given the user is on a question with a "RedirectToListAddQuestion" action enabled', function () {
+
+    before('Launch survey', function () {
+      return helpers.openQuestionnaire('test_answer_action_redirect_to_list_add_question.json');
+    });
+
+    it('When the user answers "No", Then, they should be taken to straight the list collector add question.', function () {
```

Comment:
Should this be "list collector" rather than "list collector add question"?

Reply:
Yes, it should. Fixed.